I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script_f_Interview.rpy", line 19, in script
    show Ben Interview Neutral at interview_l with dissolve
  File "game/script_f_Interview.rpy", line 19, in <module>
    show Ben Interview Neutral at interview_l with dissolve
NameError: name 'interview_l' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script_f_Interview.rpy", line 19, in script
    show Ben Interview Neutral at interview_l with dissolve
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\ast.py", line 1150, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\ast.py", line 1109, in show_imspec
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\python.py", line 1944, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\Benjamin Shu\Documents\renpy-6.99.14.1-sdk\renpy\python.py", line 1937, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script_f_Interview.rpy", line 19, in <module>
    show Ben Interview Neutral at interview_l with dissolve
NameError: name 'interview_l' is not defined

Windows-8-6.2.9200
Ren'Py 7.1.3.1092
SBU Senior Project 1.0
Tue Apr 30 18:31:34 2019
